src/App/App.tsx

```diff
@@ -73,6 +75,20 @@ function App() {
                 <br />
               </div>
             );
+
+            if (pendingTxn.metricId) {
+              const metricData = metrics.getPendingEvent(pendingTxn.metricId, true);
+              const metricType = metricData?.metricType || "unknownTxn";
+
+              metrics?.sendMetric({
+                event: `${metricType}.failed`,
```

Comment:
maybe I'm reading in long order but weird to see "failed" here, in just "metricId" if statement

Reply:
We send a metric if a metricId was passed in the transaction